torchmetrics/functional/image/psnr.py

```diff
@@ -91,7 +91,7 @@ def _psnr_update(
     return sum_squared_error, n_obs
 
 
-def psnr(
+def peak_signal_noise_ratio(
```

Comment:
need to keep the original `psnr` for back compatibility

Reply:
> @Fariborzzz could you pls resolve the conflict or shall I do it for you? rabbit

I appreciate your help, a bit confused with my first commit :rabbit2: . Hope backward compatibility works fine now.